mapie/tests/test_regression.py

```diff
@@ -502,30 +503,38 @@ def test_aggregate_with_mask_with_prefit() -> None:
     """
     Test ``_aggregate_with_mask`` in case ``cv`` is ``"prefit"``.
     """
-    mapie_reg = MapieRegressor(cv="prefit")
+    mapie_reg = MapieRegressor(LinearRegression().fit(X, y), cv="prefit")
+    mapie_reg = mapie_reg.fit(X, y)
     with pytest.raises(
         ValueError,
         match=r".*There should not be aggregation of predictions if cv is*",
     ):
-        mapie_reg._aggregate_with_mask(k, k)
+        mapie_reg.estimator_._aggregate_with_mask(k, k)
 
-    mapie_reg = MapieRegressor(agg_function="nonsense")
+
+def test_aggregate_with_mask_with_invalid_agg_function() -> None:
+    """Test ``_aggregate_with_mask`` in case ``agg_function`` is invalid."""
+    ens_reg = EnsembleRegressor(LinearRegression(), "plus",
+                                KFold(
+                                    n_splits=5, random_state=None, shuffle=True
+                                ),
+                                "nonsense", None, random_state, 0.20, False
+                                )
```

Comment:
STYLE: could you add some new lines and indents to improve the visibility/style of the code?

Reply:
Done